woocommerce/woocommerce-android
# Track simple payments separately and normal payments when select payment screen is reused

Closes: #6695


### Description

* Moves "refund" package to the "payments" package
* Renames:
    * `SIMPLE_PAYMENTS_FLOW_COMPLETED` -> `PAYMENTS_FLOW_COMPLETED`
    * `SIMPLE_PAYMENTS_FLOW_COLLECT` -> `PAYMENTS_FLOW_COLLECT`
    *  `SIMPLE_PAYMENTS_FLOW_FAILED` -> `PAYMENTS_FLOW_FAILED`
* To the aforementioned events params are added:
   * `flow: simple_payment` or `flow: order_payment`

### Testing instructions

* Run both simple payments flow and order payment flow. Click on "cash", "card" or "share payment link" on the payment selection scren
* Use logcat filtered by `Tracked`
* Notice all renamed events and newly added params
